SYCL/Assert/assert_in_simultaneous_kernels.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: linux
+// REQUIRES: TEMPORARY_DISABLED
```

Comment:
Please add comment with the disable reason. Also link to the issues is needed

Reply:
The failure was observed here: https://github.com/intel/llvm/pull/4533